example/basic/basic_main.g.dart

```diff
@@ -7,37 +7,24 @@ part of example.player;
 // **************************************************************************
 
 abstract class _$PlayerSerializer implements Serializer<Player> {
-  final AddressSerializer toAddressSerializer = new AddressSerializer();
-  final AddressSerializer fromAddressSerializer = new AddressSerializer();
+  final _addressSerializer = new AddressSerializer();
 
   Map toMap(Player model, {bool withType: false, String typeKey}) {
-    Map ret = new Map();
+    Map<String, dynamic> ret;
```

Comment:
So if model is null toMap return a null value and not an empty Map.
I think it's better to return an empty Map .

Reply:
We spoke about it but now I am not so sure,
what would happen on a backend environment when communicating with a database for exemple, there is a difference between `{}` and `null`.

@tejainece would probably know better about that

Reply:
this should also match the behavior of fromMap, that does not create the model if the map is null but return null

```
if (map is! Map) {
      return null;
    }
```